runtime-display/s2tc/spec

```diff
@@ -1,5 +1,4 @@
-VER=20151227
-REL=3
-SRCS="git::commit=f6ec862d7594e29ae80a6e49f66ad4c76cf09abc::https://github.com/divVerent/s2tc"
+VER=20210317
```

Comment:
The version should be `1.0+git20210317` according to the Styling Manual.

Reply:
Fixed.